client/js/analysisui/layoutcell.js

```diff
@@ -448,6 +453,7 @@ var SpacerCell = function(width, height, fitToGrid) {
     this.height = height;
     this.width = width;
     this.fitToGrid = fitToGrid;
+    this._visibility = true;
```

Comment:
should this be `_visible`?

visible can be `true` or `false`
visibility can be `visible` or `hidden`

but visibility can't be `true`